ivy-xref.el

```diff
@@ -46,6 +46,11 @@
   :type 'boolean
   :group 'ivy-xref)
 
+(defcustom ivy-xref-remove-properties nil
```

Comment:
Can you please rename this ivy-xref-remove-text-properties just to make it completely clear what it will do from the name?